inst/unitTests/test_shinyApp.R

```diff
@@ -0,0 +1,34 @@
+library(igvShiny)
+library(RUnit)
+library(shinytest2)
+#----------------------------------------------------------------------------------------------------
+runAppTests <- function()
+{
+  test_shinyApp
+} # runAppTests
+#----------------------------------------------------------------------------------------------------
+test_shinyApp <- function()
+{
+  message(sprintf("--- test_shinyApp"))
+  
+  options(chromote.timeout = 60)
+  shinytest2::load_app_env()
+  
+  #test_that("{shinytest2} recording: test_app", {
+    
+    app <- AppDriver$new(app_dir = system.file("app", package = "igvShiny"), name = "test_app", height = 695, width = 1235)
```

Comment:
please use the currently available demo with sth like:
```
sf <- system.file(package = "igvShiny", "demos", "igvShinyDemo.R")
app <- shinytest2::AppDriver$new(shiny::shinyAppFile(sf)
```

Reply:
done